packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.routing.js

```diff
@@ -0,0 +1,85 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.loadbalancers', {
+    url: '/loadbalancers?loadBalancerDeleteSuccess',
+    component: 'octaviaLoadBalancerListing',
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('resources')
+        .then((resources) =>
+          resources.length === 0 ? 'octavia-load-balancer.onboarding' : false,
+        ),
+    resolve: {
+      projectId: /* @ngInject */ ($transition$) =>
+        $transition$.params().projectId,
+      project: /* @ngInject */ ($http, projectId) =>
+        $http.get(`/cloud/project/${projectId}`).then(({ data }) => data),
+      breadcrumb: ($translate) => $translate.instant('octavia_load_balancers'),
```

Comment:
According to https://github.com/ovh/manager/pull/10044/files#r1343770348 comment, remove breadcrumb prefix from this state and use breadcrumb null
```suggestion
      breadcrumb: () => null,
```

Reply:
Done